config/routes_admin.yaml

```
@@ -4,3 +4,9 @@ app_albums_api:
     prefix: /admin/api
     resource: App\Controller\Admin\AlbumController
     name_prefix: app.
+
+app_additional_contact_data_api:
```

we prefix route names normally with `app.`:

```suggestion
app.additional_contact_data_api:
```

---

i think this does not affect the actual route name and we do it like this in the [`routes/sulu_admin.yaml`](https://github.com/sulu/skeleton/blob/2.x/config/routes/sulu_admin.yaml) too. but can change it if you want to. would just like to be consistent 🙂

---

yeah it has no effect on route names still would do it to be consistent.